addOns/openapi/src/test/resources/org/zaproxy/zap/extension/openapi/v3/OpenApi_defn_examples.yaml

```diff
@@ -22,6 +22,28 @@ paths:
       responses:
         '201':
           description: Created
+  # Using Examples defined in a json request body
+  /pets-with-examples:
+    post:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Pet'
+            examples:
+              fluffy-example:
+                description: a pet called Big Fluffy
+                value:
+                  age: 6
+                  name: Big Fluffy
+              fawkes-example:
+                description: a pet called Small Fawkes
```

Comment:
Not that it matters for the code/tests but it's actually called `Small Fluffy`.

Reply:
updated.